<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterTableView</class>
 <widget class="QWidget" name="ParameterTableView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetWhiteBase" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>26</number>
      </property>
      <property name="topMargin">
       <number>26</number>
      </property>
      <property name="rightMargin">
       <number>26</number>
      </property>
      <property name="bottomMargin">
       <number>26</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="CustomCheckBox" name="SampleCheckBox">
          <property name="text">
           <string>&quot;PICKED %n SAMPLES&quot;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomCheckBox" name="PeakCheckBox">
          <property name="text">
           <string>&quot;PICKED %n PEAKS&quot;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="ParametersLabel">
          <property name="text">
           <string>Parameters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ParametersComboBox">
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Control::EventTableView" name="tableView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ui/controls/customcheckbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Control::EventTableView</class>
   <extends>QTableView</extends>
   <header>ui/itemviews/event_table_view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
